media-sound/mpd/files/mpd-0.20.21.logrotate

```diff
@@ -4,6 +4,11 @@
 	delaycompress
```

Comment:
Please add this only to the latest version, also please change a revision of the ebuild so users can get the updated during explicit uodate request.

Also `--quiet` option has to remain in the else branch, why did you change this?

And I would rewrite this like:

```
	postrotate
		if type -p systemctl 2>&1 1>/dev/null && systemctl is-system-running 2>&1 1>/dev/null ; then
			systemctl kill -s HUP mpd.service
		else
			/etc/init.d/mpd --quiet reload
		fi
	endscript
```

Reply:
Forgot to keep `--quiet` since the old versions that use init don't have that. I have made the changes requested.